src/js/import.js

```diff
@@ -195,8 +220,17 @@
         $(".import-extraHeaders").each(function() {
           selected.push($(this).val());
         });
-        $(".import-leave").each(function() {
-          selected.push($(this).val());
+        $(".import-intervention").each(function() {
+         selected.push($(this).val());
+        });
+        $(".import-effect").each(function() {
+         selected.push($(this).val());
+        });
```

Comment:
According to http://kangax.github.io/compat-table/es6/#test-Array_static_methods_Array.from,_array-like_objects it seems quite safe to use `Array.from` on array-like objects (e.g. `NodeList`)